app/src/main/java/io/github/hidroh/materialistic/WebFragment.java

```diff
@@ -572,4 +619,62 @@ public void onError(String errorMessage) {
             // do nothing
         }
     }
+
+    static class PdfAndroidJavascriptBridge {
+        Context mContext;
+        File mFile;
+        @Nullable RandomAccessFile mRandomAccessFile;
+
+        PdfAndroidJavascriptBridge(Context context, String filePath) {
+            mContext = context;
+            mFile = new File(filePath);
+            try {
+                mRandomAccessFile = new RandomAccessFile(mFile, "r");
```

Comment:
Do you think it's better if we defer this instantiation to beginning of `getChunk()`?

Reply:
Probably won't make much difference, since it gonna happen several milliseconds later :) But okay, I'lll move it.